DTIPrep/DTIPrepRegister.pl

```diff
@@ -181,7 +191,7 @@
                                                     "DTIPrep");
 if (!$registeredXMLprotocolID) {
     print LOG "\nERROR: no XML protocol file was registered in the database\n";
-    exit 0;
+    exit $NeuroDB::ExitCodes::XML_PROTOCOL_INSERT_FAILURE;
```

Comment:
mmm, just curious, isn't exit 0 reserved for Success???

Reply:
Indeed, which is why it was a mistake in the past to exit 0 when the file registration failed. I created the script and did not know the meaning of the exit code at the time, so I would not really rely on what I wrote at that time ;).